packages/react/toolbar/src/Toolbar.tsx

```diff
@@ -14,57 +14,40 @@ import type * as Polymorphic from '@radix-ui/react-polymorphic';
  * -----------------------------------------------------------------------------------------------*/
 
 const TOOLBAR_NAME = 'Toolbar';
+const TOOLBAR_DEFAULT_TAG = 'div';
 
-type Orientation = React.AriaAttributes['aria-orientation'];
-type RovingFocusGroupProps = React.ComponentProps<typeof RovingFocusGroup>;
-
-type ToolbarOwnProps = Polymorphic.Merge<
-  Polymorphic.OwnProps<typeof Primitive>,
-  {
-    /**
-     * The orientation of the toolbar.
-     * Mainly so arrow navigation is done accordingly (left & right vs. up & down)
-     *
-     * @defaultValue horizontal
-     */
-    orientation?: RovingFocusGroupProps['orientation'];
-    /**
-     * The direction of navigation between toolbar items.
-     *
-     * @defaultValue ltr
-     */
-    dir?: RovingFocusGroupProps['dir'];
-    /**
-     * Whether keyboard navigation should loop focus
-     *
-     * @defaultValue true
-     */
-    loop?: RovingFocusGroupProps['loop'];
-  }
+type ToolbarOwnProps = Omit<
+  Polymorphic.OwnProps<typeof RovingFocusGroup>,
+  'currentTabStopId' | 'defaultCurrentTabStopId' | 'onCurrentTabStopIdChange' | 'onEntryFocus'
 >;
-
 type ToolbarPrimitive = Polymorphic.ForwardRefComponent<
   Polymorphic.IntrinsicElement<typeof Primitive>,
```

Comment:
It's not rendering `Primitive` anymore so `typeof TOOLBAR_DEFAULT_TAG`

Reply:
Yep, missed that one!